blocks/firefly-footer/firefly-footer.css

```diff
@@ -0,0 +1,69 @@
+.firefly-footer {
+  align-self: center;
+  padding: 35px 24px 0;
+  font-size: 11px;
+  line-height: normal;
+}
+
+.firefly-footer > div{
+  border-top: solid 1px rgba(213,213,213);
+  margin-top: 10px;
+}
+
+.firefly-footer > div:nth-child(1){
+  padding-top: 8px;
+  padding-bottom: 8px;
+}
+
+.firefly-footer > div:first-of-type p{
+  text-decoration: none;
+  color: inherit;
+  font-style: italic;
+  text-align: left;
+  margin: 0;
+}
+
+.firefly-footer img[src$=".svg"] {
+  height: 16px;
+}
+
+.firefly-footer .footer-links ul{
+  list-style-type: none;
+}
+
+.firefly-footer .footer-links > div > div:nth-child(2) ul{
+  display: flex;
+  flex-flow: wrap;
+  gap: 8px;
+  padding: inherit;
+  margin-bottom: 20px;
```

Comment:
this margin-bottom, can it be on the overall footer rather than here? it is causing all the items in the footer to not be aligned vertically center.